src/platform/user/profile/vap-svc/tests/components/VAPServiceEditModalErrorMessage.unit.spec.jsx

```diff
@@ -32,4 +28,27 @@ describe('<VAPServiceEditModalErrorMessage />', () => {
     );
     wrapper.unmount();
   });
+  it('shows the correct error message when there is an invalid phone area code', () => {
+    const invalidPhoneError = {
+      errors: [
+        {
+          title: 'Area Code Pattern',
+          detail: 'Phone area code pattern must match "[0-9]+"',
+          code: 'VET360_PHON126',
+          source: 'Vet360::ContactInformation::Service',
+          status: '400',
+        },
+      ],
+    };
+    const wrapper = shallow(
+      <VAPServiceEditModalErrorMessage error={invalidPhoneError} title="" />,
+    );
+    const alert = wrapper.find('AlertBox');
+    const alertContentText = alert?.prop('content')?.props?.children?.props
```

Comment:
We shouldn't need all these `?`. If the preceding line doesn't set `alert` properly we want this to blow up. And also OMG `alert.prop('content').props.children.props.children` is why Enzyme needs to die in a fire and why FE developers hate writing tests.

Reply:
LOL, I just copy/pasted & modified the test above it.